python/pip.bzl

```diff
@@ -165,6 +166,31 @@ def pip_parse(requirements_lock, name = "pip_parsed_deps", **kwargs):
         name = "flake8",
         actual = entry_point("flake8"),
     )
+
+    ## Vendoring the requirements.bzl file
```

Comment:
This commentary is place in the middle of a Starlark codeblock. The gen'd `.md` has syntax highlighting on it. 

I think the above codeblock just needs to be closed before starting this commentary. It's currently closed at https://github.com/bazelbuild/rules_python/pull/655/files#diff-b92f94a67df862cde9714ff224afa676e39fe02da4fd4c6993daf684fbbc7cbdR194

Reply:
oops! thanks